src/components/Chat/Chat.module.css

```diff
@@ -0,0 +1,61 @@
+.chatRoom {
+    height: 100%;
+    font-size: 1em;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    background-color: white;
+    border: 2px solid black;
+    border-radius: 1em;
+    margin: 0em 0.5em 0.5em 0.5em;
+    letter-spacing: 0.2em;
+}
+
+.chatBox {
+    font-size: 1em;
+    height: 80%;
```

Comment:
I think you can remove the `height`s that you have in this file as you are using flex so you can use flex-grow and margins to get things looking right